core/embed/models/T3W1/boards/t3w1-unix.h

```diff
@@ -1,11 +1,8 @@
 #ifndef BOARDS_T3W1_UNIX_H
 #define BOARDS_T3W1_UNIX_H
 
-#define MAX_DISPLAY_RESX 380
-#define MAX_DISPLAY_RESY 520
 #define DISPLAY_RESX 380
```

Comment:
these should be removed

Reply:
fixed, see 986f6b804e537be653745bb9758e6f44c77193bf